examples/storage-gke.yaml

```diff
@@ -38,9 +38,33 @@ deployment_groups:
         - range_name: services
           ip_cidr_range: 10.0.32.0/20
 
+  - id: gke_service_account
```

Comment:
Isn't `gcp_service_account` more descriptive? GKE service accounts are to manage k8s resources while the account we need is GCP IAM sa.

Reply:
please ignore